### PR TITLE
[Ignore for now]Add cookie pop-up

### PR DESCRIPTION
**Pull request by theodorsm**
Add annoying cookie consent pop-up. 

image

### CODE REVIEW

File: app/routes/app/AppRoute.js [hunk 1]
```diff
@@ -141,6 +142,12 @@ class App extends PureComponent<AppProps> {
 
           <PhotoUploadStatus />
 
+          <CookieConsent
+            content="Når du besøker abakus.no installerer vi cookies som er nødvendig for at du skal kunne benytte deg av nettsidene våre."
```
Comment (AdrianAndersen) [inline app/routes/app/AppRoute.js:146]:
```suggestion
            content="Når du besøker abakus.no, installerer vi cookies som er nødvendig for at du skal kunne benytte deg av nettsidene våre."
```

File: app/components/CookieConsenter/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+const CookieConsenter = ({ content, buttonText, link }: Props) => {
+  return (
+    <CookieConsent
+      style={{ background: '#bc1818' }}
+      buttonStyle={{ background: 'white', borderRadius: '5px' }}
+      buttonText={buttonText}
+    >
+      {content + ' '}
+      {link ? (
```
Comment (LudvigHz) [inline app/components/CookieConsenter/index.js:22]: `link && (...)`?

**Review by LudvigHz — APPROVED** [review]
lgtm

**Review by AdrianAndersen — COMMENTED** [review]
Looking good. I would recommend to use --css-variables on your colors to make it more uniform. (For instance **var(--lego-red)** instead of **#bc1818**) Make sure to work out the whether the styles sheet should be used or not. I also added a grammar-suggestion. GDPR-compliance here we come. :smile:

File: app/routes/app/AppRoute.js (hunk 1 above)
Comment (theodorsm) [reply app/routes/app/AppRoute.js:146]: Thanks for the review. I will correct the grammar, but it is not easy to use css-variables as the CookieConset component needs inline styling, so it's not possible to use css classes.

Comment (AdrianAndersen) [reply app/routes/app/AppRoute.js]: Ait, no worries :+1:

**Review by AdrianAndersen — APPROVED** [review]

**Comment by odinuge** [comment]
Can we avoid using cookies for un-authed users instead of this? And instead store the answer for each user in the db so that folks only need to agree once.

**Comment by theodorsm** [comment]
Need to migrate stripe: https://github.com/stripe/react-stripe-js/blob/master/docs/migrating.md

**Comment by theodorsm** [comment]
We are thinking about making a privacy page for users later.

**Comment by theodorsm** [comment]
Screenshots: 
image

image

**Comment by theodorsm** [comment]
The website is not unsuable for users without consenting for cookies. If you take remove your consent, the banner will show again.
image

**Comment by LudvigHz** [comment]
Closing this in favor of #1985

**Pull request closed without merging**